central/test-framework/org.jboss.tools.central.reddeer/src/org/jboss/tools/central/reddeer/wizards/JBossCentralProjectWizardLauncherApplication.java

```diff
@@ -0,0 +1,231 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc.
```

Comment:
Please update copyright year.

Reply:
Looks like this is duplicate file I forget to delete when moving the file. Deleted. (There are no dependencies from other repositories to this file).

Reply:
In original file the year is updated.